src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IAnonymousFunctionExpression.cs

```diff
@@ -201,22 +201,22 @@ static void F()
 }
 ";
             string expectedOperationTree = @"
-IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null, IsInvalid) (Syntax: 'var x = () => F();')
```

Comment:
Unless I'm missing something, the only changes in these tests are now whitespace. Consider reverting this file to avoid unnecessary changes.